config.go

```diff
@@ -121,6 +131,32 @@ type CORSConfig struct {
 	//
 	// Added in v4.2.0.
 	AllowAllOrigins bool
+
+	// AllowOrigins enables CORS and allows the list of origins in the
+	// HTTP request origins when set. Practically speaking, this
+	// results in the HTTP header "AllowOrigins".
+	//
+	// Added in v5.0.0.
+	AllowOrigins string
+}
+
+// OICDConfig holds settings for the HTTP server's OIDC access token validation settings.
+type OICDConfig struct {
+
+	Enable bool
+
```

Comment:
nit: Suggest `Enabled` instead of `Enable`.

Reply:
I prefer enable.